mods/gen/rules/structures.yaml

```diff
@@ -3229,24 +3231,10 @@ building.stinger_site:
 	WithIdleOverlay@CAMO:
 		Sequence: idle-camo
 		Palette: cloak
-	MobSpawnerMaster@A:
-		Actors: infantry.stinger_soldier
-		SpawnOffset: 511,511,0
-		RespawnTicks: 750
-		ExitByBudding: false
-		AggregateHealth: false
-	MobSpawnerMaster@B:
-		Actors: infantry.stinger_soldier
-		SpawnOffset: -511,-511,0
-		RespawnTicks: 750
-		ExitByBudding: false
-		AggregateHealth: false
-	MobSpawnerMaster@C:
-		Actors: infantry.stinger_soldier
-		SpawnOffset: -511,511,0
-		RespawnTicks: 750
-		ExitByBudding: false
-		AggregateHealth: false
+	DroneSpawnerMaster:
+		SpawnOffset: 511,511,0, -511,-511,0, -511,511,0
+		Actors: infantry.stinger_soldier, infantry.stinger_soldier, infantry.stinger_soldier
+		RespawnTicks: 400
```

Comment:
Can you update the fake one too, if you are changing the trait here?

Reply:
Sure

Reply:
Now added